recipes/kwant/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "kwant" %}
+{% set version = "1.1.2" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://gitlab.kwant-project.org/kwant/kwant/repository/archive.tar.gz?ref=v{{ version }}
+  sha256: a47544a8416b19a56da3d511610c9e09ed3bf84f66461d9f95f0f6fefebd0179
+
+build:
+  skip: true  # [win]
+  skip: true  # [py3k] 
```

Comment:
Fixed in 82ea10cfd04e0355b19f591efa1106c2c19492b2. There is no version that combines both Python 2 and 3 simultaneously.